chapters/annotations.tex

```diff
@@ -1376,16 +1393,16 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
 \end{lstlisting}
 \end{enumerate}
 
-\emph{These rules also apply if the connectors and/or components are
-defined in {superclass}. Example:} \emph{Assume that step1 is defined in
-superclass CompositeStep with 3 connections, and a new connection is
-performed in a subclass. The result is:}
+These rules also apply if the connectors and/or components are
+defined in \lstinline!superclass!. Example: Assume that \lstinline!step1! is defined in
```

Comment:
I don't think that superclass should be lstinline, as it isn't a specific class called superclass, but just any superclass.
(Same on following lines).

Reply:
Done.